controllers/users.js

```diff
@@ -1,53 +1,56 @@
 const bcrypt = require('bcryptjs');
 const jwt = require('jsonwebtoken');
 
+const NotFoundError = require('../errors/error_not_found');
+const Error500 = require('../errors/error_500');
+const Error401 = require('../errors/error_Auth');
 const User = require('../models/user');
-/*
+
 const { JWT_SECRET } = require('../secret.js');
-const token = jwt.sign({ _id: user._id }, JWT_SECRET, { expiresIn: '7d' });
-*/
+
 // отправим токен, браузер сохранит его в куках
 
 module.exports.getAllUsers = (req, res) => {
   User.find({})
     .then((user) => {
       if (user.length === 0) {
-        return res.status(404).send({ message: 'База данных user пуста! ' });
+        return (new NotFoundError({ message: 'База данных user пуста! ' }));
       }
       return res.send({ data: user });
     })
-    .catch((error) => res.status(500).send({ message: error.message }));
+    .catch((new Error500('На сервере произошла ошибка')));
```

Comment:
Нужно исправить: ошибка не обрабатывается централизованным обработчиком.
Централизованная обработка ошибок запустится только если будет вызван `next`:
```suggestion
    .catch(next);
```

Только нужно не забыть объявить `next` здесь:
https://github.com/MaximMaslovVik/15_sprint/blob/2ad864e57b00f8564f3658374d392b861c3b10fb/controllers/users.js#L13